homeworks/12-mars-viewer-mobx/src/components/loadPanel/LoadPanel.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {
+  ChangeEventHandler, MouseEventHandler, useCallback, useState,
+} from 'react';
+import { observer } from 'mobx-react-lite';
+import { fetchSols } from '../../api/fetchAPI';
+import menuStore from '../../store/menuStore';
+import solsStore from '../../store/solsStore';
+import style from './loadPanel.module.css';
+
+export const LoadPanel = observer(() => {
+  const { activeMenu } = menuStore;
+  const { fetchSolsStart, fetchSolsSuccess, fetchSolsError } = solsStore;
+  const [selectedSol, setSelectedSol] = useState('');
+  const onClickHandle: MouseEventHandler<HTMLButtonElement> = async () => {
+    fetchSolsStart();
```

Comment:
This logic is more preferable for models, your component has no reason to handle asynchronous code